collection-scripts/gather

```diff
@@ -45,13 +45,17 @@ oc adm inspect --dest-dir must-gather "${named_resources[@]}"
 group_resources_text=$(IFS=, ; echo "${group_resources[*]}")
 oc adm inspect --dest-dir must-gather "${group_resources_text}"
 
+# Gather monitoring data from the cluster
+/usr/bin/gather_monitoring
+
 # Gather optional operator resources from all namespaces
 /usr/bin/gather_olm
 
 # Gather etcd information
 /usr/bin/gather_etcd
 
 # Gather Validation and Mutating Admission Webhook Objects
+# TODO the kube-apiserver operator should list these as related resources
```

Comment:
If this is the case, should we merge this? We have pushed other teams in the past to do this over adding resource collectors in this way.

Reply:
> If this is the case, should we merge this? We have pushed other teams in the past to do this over adding resource collectors in this way.

this PR doesn't make it worse.  I happened to catch this on the way though as I integrated and wanted to suggest a vision.

Reply:
+1 who can look at that as a bug/rfe (in 4.8/4.9)?